src/script/components/Meeting/MeetingList/MeetingList.tsx

```diff
@@ -126,10 +83,14 @@ export const MeetingList = () => {
   }
 
   if (isNextTab) {
-    // Next tab
     content = hasMeetingsToday ? (
```

Comment:
Better use early return with regular if instead of the shorthand. It makes the code more readable